Update config.mts to fix links & add Delete post, typing, and ulist (#16)
meower-media/docs: docs/api/cloudlink/packets/ulist.md

# `ulist`

This packet provides a string of online users separated by ";"
## Payload

A string of users separated by ";"
```
user1;user2;user3;
```

## Usage

> ::: code-group

Reviewer: 1. Is this section necessary at all? 
2. All the code blocks should remove the last element as well as the ulist puts a semicolon at the end

Author: 1. Probably not, I can remove it.

```ts [TypeScript]
// Converts the online userlist into a array of strings
// val is the string of users given by the websocket

> let onlineUsers: string[] = val.split(";");

Reviewer: The JS code works in Typescript as well and is more idiomatic

Author: Ok, I will merge them into one.

# `ulist`

> This packet provides a string of online users separated by ";"

Reviewer 2: ```suggestion
This packet contains a comma-separated list of all online users

```

Author: Ok, I will update that.
